apps/site/lib/site_web/controllers/schedule/line/diagram_helpers.ex

```diff
@@ -0,0 +1,326 @@
+defmodule SiteWeb.ScheduleController.Line.DiagramHelpers do
```

Comment:
@meagonqz is this file basically a cut-and-paste of the big chunk of code that formerly was in `ScheduleController.Line`, or did you also make more significant changes in the process?

Reply:
Yes, this code was just moved over into it's own module so it could be shared (and minimize the size and complexity of that Line `plug`)

Reply:
that's what I was hoping to hear